src/partials/SiteAlert.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  import {
+    InlineNotification,
+    NotificationActionButton,
+    Link,
+  } from "carbon-components-svelte";
+</script>
+
+<style>
+  .alert :global(.bx--inline-notification) {
+    margin: 0;
+    max-width: 100%;
+  }
+</style>
+
+<div class="alert">
+  <InlineNotification
+    lowContrast
+    hideCloseButton
+    kind="info"
+    title="This is a preview of the new Cal-Adapt website."
+    subtitle="Return to cal-adapt.org to access all climate tools and data."
+  >
+    <div slot="actions">
+      <NotificationActionButton>
+        <Link href="https://cal-adapt.org/">cal-adapt.org</Link>
```

Comment:
Maybe we can say "Return to Cal-Adapt" or "Go to Cal Adapt" here for the link text? I think ideally it would help to make the subtitle prop in the `InlineNotification` component a link but that doesn't seem like an option with this component.

Reply:
The `subtitle` prop seems to accept only text. That would have been ideal. I will update the link text.

Reply:
Fxed via https://github.com/berkeley-gif/caladapt-website-2021/pull/64/commits/79841d5f0932e6a763e25560e9495a3820b8fe5a